app/Bundle.php

```diff
@@ -53,9 +57,100 @@ class Bundle extends Model
     protected $dates = [
         'created_at',
         'updated_at',
-        'allocated_at'  // When it was handed out.
+        'disbursed_at'  // When it was handed out.
     ];
 
+    /**
+     * Refactored function that works out if we broke anything then adds vouchers.
+     *
+     * @param Collection $vouchers
+     * @param array $codes
+     * @param Bundle|null $bundle
+     * @return array
```

Comment:
Is the return always an array of errors (which could be empty - signifying success or maybe that nothing happened)?

Reply:
If errors is empty, there are no errors, so success.